<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditOrderDialog</class>
 <widget class="QDialog" name="EditOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Order</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4"/>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit">
      <property name="placeholderText">
       <string>Имя цели</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="descriptionEdit">
      <property name="placeholderText">
       <string>Описание цели</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </item>
    <item>
     <widget class="QComboBox" name="statusComboBox">
      <item>
       <property name="text">
        <string>Ожидается</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>В процессе</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Завершено</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Просрочено</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="startDateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="endDateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>330</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Удалить заказ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="photoLabel">
      <property name="text">
       <string>Photo:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadPhotoButton">
      <property name="text">
       <string>Load Photo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
